torch/lib/c10d/Utils.hpp

```diff
@@ -536,8 +566,13 @@ void sendBytes(
 
   while (bytesToSend > 0) {
     ssize_t bytesSent;
+#ifdef _WIN32
```

Comment:
Can these be removed? I believe that `::send` will work on Windows. And `(const char*)currentBytes` should work on Linux.

Reply:
::send works. And on Linux, can not sure (const char*) whether works, let me try it.